Package.swift

```diff
@@ -8,14 +8,17 @@ let package = Package(
     name: "danger-swift",
     products: [
         .library(name: "Danger", type: .dynamic, targets: ["Danger"]),
-        .executable(name: "danger-swift", targets: ["Runner"])
+        .executable(name: "danger-swift", targets: ["Runner"]),
     ],
     dependencies: [
         .package(url: "https://github.com/JohnSundell/Marathon.git", from: "3.1.0"),
         .package(url: "https://github.com/JohnSundell/ShellOut.git", from: "2.1.0"),
         .package(url: "https://github.com/nerdishbynature/octokit.swift", from: "0.9.0"),
         // Dev dependencies
-        .package(url: "https://github.com/eneko/SourceDocs.git", from: "0.5.1"),
+        .package(url: "https://github.com/eneko/SourceDocs.git", from: "0.5.1"), // dev
+        .package(url: "https://github.com/orta/Komondor.git", from: "1.0.0"), // dev
+        .package(url: "https://github.com/nicklockwood/SwiftFormat.git", from: "0.35.8"), // dev
+        .package(url: "https://github.com/Realm/SwiftLint.git", from: "0.28.1"), // dev
```

Comment:
If a dep has "// dev" after it, then it will be commented out

Reply:
Cool, could it just comment everything that is after `// Dev dependencies`?

Reply:
something like: everything that starts with `.package(url:` after `// Dev dependencies`